styles.css

```diff
@@ -1 +1,84 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+div {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 50px 190px 100px auto 50px;
+    padding: 16px;
+    background: lightpink;
+    grid-template-areas:
+        "item1 item3 item3"
+        "item2 item3 item3"
+        "item2 item5 item4"
+        "item2 item5 ."
+        "item6 item6 item6";
+    row-gap: 16px;
+    column-gap: 16px;
+}
+
+#item1 {
+    background-color: crimson;
+    grid-area: item1;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+#item2 {
+    background-color: crimson;
+    grid-area: item2;
+    padding: 10px;
+}
+
+#item3 {
+    background-color: crimson;
+    grid-area: item3;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+#item4 {
+    background-color: crimson;
+    grid-area: item4;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+#item5 {
+    background-color: crimson;
+    grid-area: item5;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+#item6 {
+    background-color: crimson;
+    grid-area: item6;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+}
+
+@media (max-width: 600px) {
```

Comment:
Hier is niets mis mee, maar Nova benoemde in de les dat er eigenlijk altijd met de min-width gewerkt wordt, probeer jezelf dit gelijk aan te leren.